docs/src/components/page-parts/platform-integrations/QuasarIntegrations.vue

```diff
@@ -19,54 +19,84 @@
     <h2
       class="landing-heading landing-heading--medium q-mb-lg normal-line-height"
     >We have an impressive gear</h2>
-    <div class="integration-options row justify-center q-gutter-lg q-mb-xl">
+    <div class="row justify-center q-gutter-sm q-mb-xl">
       <q-card
```

Comment:
Wrap these with a `router-link` too, or it won't work when clicked with middle button of the mouse